test/k8sT/Services.go

```diff
@@ -160,6 +162,8 @@ var _ = Describe("K8sServicesTest", func() {
 		BeforeAll(func() {
 			res := kubectl.Apply(demoYAML)
```

Comment:
Perhaps refactor this and the above on line 121-124 into a separate function to reduce code duplication.

Reply:
I started down this road but I ended up with more code than before. It's trivial to factor out the code itself but then I noticed that there is a cleanup section, but it is distinct between the two contexts. It felt a little odd to wrap the creation but leave the deletion open. (There was also the oddness of the BeforeAll taking the wrapper directly, and that looked weird compared to all the other things). Since these were different contexts (and different AfterAll vs AfterEach) I left it as it is.

Reply:
Makes sense, thanks for explaining!